.gitattributes

```diff
@@ -1,3 +1,3 @@
-* text=auto
+* text eol=lf
```

Comment:
Partly because of the .editorconfig and mostly because when I cloned and ran the test, I got an error for every line telling me it encountered a CRLF but expected a LF. I did not have an editor config plugin installed at first.